client/src/modules/stock/inventory-adjustment/inventory-adjustment.js

```diff
@@ -129,8 +129,9 @@ function StockInventoryAdjustmentController(
   function startup() {
     vm.movement = {
       date : new Date(),
-      entity : {},
```

Comment:
Why was this removed?

Reply:
Good question.  It was never used or needed.  I think it is copy/paste from somewhere where it was needed.